src/components/media-loader.js

```diff
@@ -121,3 +125,54 @@ AFRAME.registerComponent("media-loader", {
     }
   }
 });
+
+AFRAME.registerComponent("media-pager", {
```

Comment:
Are you purposefully putting the `media-loader` and the `media-pager` in the same file? That is fine if you are, just wondering if this is an accident.

Reply:
I did yeah, though I know I tend to be the only one that likes to do this.

Reply:
The end of an AFRAME component feels like an arbitrary boundary point to split up files so I don't mind it.